Library/Homebrew/language/node.rb

```diff
@@ -85,5 +85,30 @@ def self.local_npm_install_args
         --#{npm_cache_config}
       ]
     end
+
+    # Mixin module for {Formula} adding shebang rewrite features.
+    module Shebang
+      module_function
+
+      # @private
+      sig { params(node_path: T.any(String, Pathname)).returns(Utils::Shebang::RewriteInfo) }
+      def node_shebang_rewrite_info(node_path)
+        Utils::Shebang::RewriteInfo.new(
+          %r{^#! ?/usr/bin/(?:env )?node( |$)},
+          21, # the length of "#! /usr/bin/env node "
```

Comment:
```suggestion
          "#! /usr/bin/env node ".length,
```

could be made a constant, too.